packages/jest-cli/src/watch.js

```diff
@@ -49,6 +49,18 @@ const INTERNAL_PLUGINS = [
   QuitPlugin,
 ];
 
+const RESERVED_KEY_PLUGINS = new Map([
+  [
+    UpdateSnapshotsPlugin,
+    {forbiddenOverwriteMessage: 'updating snapshots ballpark', key: 'u'},
```

Comment:
Should we change this to just `updating snapshots` instead of `update snapshots ballpark`?

Reply:
Trimmed.